default/scripting/species/common/industry.macros

```diff
@@ -5,15 +5,25 @@ BASIC_INDUSTRY
 '''        EffectsGroup
             scope = Source
             activation = And [
-                Source
                 Planet
                 TargetIndustry low = 0
                 Focus type = "FOCUS_INDUSTRY"
             ]
             accountinglabel = "FOCUS_INDUSTRY_LABEL"
-            priority = [[EARLY_PRIORITY]]
+            priority = [[DEFAULT_PRIORITY]]
```

Comment:
That'd be because I forgot, sorry, friend visiting. [f259042](https://github.com/freeorion/freeorion/commit/f2590428b3a716ec40e547953972b195f2d22735)

Probably ought to have some comments somewhere explaining priorities better, ideally the default multipliers wouldn't be at default priority as any new effects may clash, ah well, hindsite, something to fix in future.